gateways/ossm-about-gateways.adoc

```diff
@@ -6,7 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[] 
 
-A gateway is a standalone Envoy proxy deployment and an associated Kubernetes service operating at the edge of a service mesh. You can configure a gateway to provide fine-grained control over the traffic that enters or leaves the mesh. In {SMProductName}, you install gateways using gateway injection.
+[role="_abstract"]
+A gateway is a standalone Envoy proxy deployment and an associated Kubernetes service operating at the edge of a service mesh. You can configure a gateway to give fine-grained control over the traffic that enters or leaves the mesh. In {SMProductName}, you install gateways by using gateway injection.
```

Comment:
CC: Jamie

```suggestion
A gateway is a standalone Envoy proxy deployment and an associated Kubernetes service operating at the edge of a service mesh. You can configure a gateway to give fine-grained control over the traffic that enters or leaves the mesh. In {SMProductName}, you install gateways by using gateway injection or via the Gateway API.
```

Reply:
@sridhargaddam I wasn't sure what the question here is? The addition looks good to me.